cfg/DB/func/get_log.ddl

```diff
@@ -0,0 +1,44 @@
+CREATE OR REPLACE FUNCTION get_log_test(Id integer, isCand bool)
+    RETURNS table(
```

Comment:
will be better use json output

Reply:
its an direct output for front, I can change, but it will affect front